libraries/AP_HAL/RCInput.h

```diff
@@ -34,7 +34,7 @@ class AP_HAL::RCInput {
 
     /* get receiver based RSSI if available. -1 for unknown, 0 for no link, 255 for maximum link */
     virtual int16_t get_rssi(void) { return -1; }
-
+    virtual int16_t get_quality(void) { return -1; }
```

Comment:
I think you should call this get_link_quality() and change the other accessors to match

Reply:
Also not sure int16_t is right here since its a percentage. Either int8_t or float I think.

Reply:
will do...is it possible for you to test with a CRSF setup to see if you can access the link quality?

Reply:
done....I think it should remain int16 in case another protocol uses that?? like we do for RSSI
add rx_link in front of most quality variables/functions